cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js

```diff
@@ -19,14 +19,17 @@ function setInnerWidth(width) {
  * so some elements won't be expanded by default.
  */
 function beforeExpandableTransitionInit() {
-  document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS ).forEach( el => {
+  const nodeList = document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS );
+
+  // IE11 lacks NodeList.forEach
+  [].forEach.call(nodeList, el => {
```

Comment:
@Scotchester I don't have IE11 to test, but this appears the go-to workaround for this.

Reply:
Sorry, I'll fix linting.